modules/jjwg_Maps/views/view.config.php

```diff
@@ -423,6 +423,10 @@ function display() {
             title='' tabindex='140' size="25" maxlength="255">
             &nbsp; <?php echo $GLOBALS['mod_strings']['LBL_CONFIG_DEFAULT']; ?>
                 <?php echo htmlspecialchars($GLOBALS['jjwg_config_defaults']['geocoding_api_secret']) ?>
+            <br/>
+            <a href='https://developers.google.com/maps/documentation/geocoding/get-api-key' target='_blank'>
```

Comment:
Can you configure this to enable translations? 
See related PR https://github.com/salesagility/SuiteCRM/pull/2536